### PR TITLE
Add navigation

### PR DESCRIPTION
**Pull request by EdgarB**
## Description
Add navigation bar.

## How has this been tested?

Please mark the tests that you ran to verify your changes. If difficult to test, consider providing instructions so reviewers can test.

- [ ] Manual testing
- [ ] System tests
- [ ] Unit tests
- [ ] None

## Checklist

- [ ] CI pipeline is passing
- [ ] My code follows the conventions of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (if applicable)
- [ ] I have added seed data to the database (if applicable)

## Release tasks

Add any tasks that need to be done before/after the release of this feature.

## Screenshots/Loom

This section is relevant in case we want to share progress with the team, otherwise, it can be omitted.

### CODE REVIEW

File: app/helpers/navigation_helper.rb [hunk 1]
```diff
@@ -1,4 +1,22 @@
...
+  NAVIGATION_LINKS = {
+    agenda: "/",
+    schedule: "/",
+    profile: "/profile",
+    notifications: "/",
+    about: "/"
+  }
+
+  def nav_icon_class_for(item)
+    return "fill-red w-6 h-6" if current_page?(NAVIGATION_LINKS[item])
```
Comment (LuigiR0jas) [inline app/helpers/navigation_helper.rb:11]: I think using [`class_names`](https://til.hashrocket.com/posts/yitpbvf6dw-easy-conditional-style-class-names-in-rails) is a better approach

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -32,6 +32,7 @@
       <% if user_signed_in? %>
         <%= button_to "Log out", session_path, method: :delete %>
       <% end %>
+      <%= render partial: "/layouts/bottom_navbar" %>
```
Comment (LuigiR0jas) [inline app/views/layouts/application.html.erb:35]: The navbar should only be visible if the user is logged in

File: app/views/profiles/show.html.erb [hunk 3]
```diff
@@ -1,4 +1,4 @@
-<div class="flex flex-col items-center bg-gray-8 relative min-h-screen justify-between ">
+<div class="flex flex-col items-center bg-gray-8 relative min-h-[calc(100vh-138px)] justify-between ">
```
Comment (LuigiR0jas) [inline app/views/profiles/show.html.erb:1]: Let's preferably use `flex-1` to make the page take all the available space

File: app/assets/images/avatar_no_fill.svg
Comment (LuigiR0jas) [inline app/assets/images/avatar_no_fill.svg:1]: The icons in the design recently changed. Can you update them please?

**Review by LuigiR0jas — COMMENTED** [review]
Looks good so far!

Another thing I noticed is that the navbar should stay always at the bottom even if there's no content pushing it down.

File: app/helpers/navigation_helper.rb (hunk 1 above)
Comment (EdgarB) [reply app/helpers/navigation_helper.rb]: I updated the approach, I think this works best. Let me know what you think 👍

**Review by LuigiR0jas — APPROVED** [review]

**Pull request merged**